docs/install.rst

```diff
@@ -0,0 +1 @@
+:orphan:
```

Comment:
What's up with this file?  A file that's `:orphan:` and with no content doesn't seem like it does anything.